backend/src/main/kotlin/pl/poznan/put/tsd/planningpoker/backend/resources/MessageHandler.kt

```diff
@@ -101,17 +101,17 @@ class MessageHandler(val gamesService: GamesService) :
     }
 
     private fun Game.findPlayer(username: String) = either {
-        val player = this@findPlayer.players[username]
-        ensureNotNull(player) { ValidationError.InvalidUsername }
-        player
+        val playerGameConnection = gamesService.playerGameConnections.filter { it.game.id == id && it.player.name == username }
```

Comment:
this could be `gamesService.playerGameConnections.first { it.game.id == id && it.player.name == username }`